roles/arteria-sequencing-report-ws/tasks/install.yml

```diff
@@ -0,0 +1,36 @@
+
+- name: create virtual python env for sequencing report service
+  shell: "conda create --name {{ arteria_service_name }} python=3.8"
+  args: 
+    creates: "{{ arteria_service_env_root }}"
+
+- name: get sequencing report service from git
+  git:
+    repo: "{{ seqreport_service_repo }}"
+    dest: "{{ arteria_service_src_path }}"
+    version: "{{ seqreport_service_version }}"
+
+- name: install sequencing report service requirements
+  pip:
+    requirements: "{{ arteria_service_src_path }}/requirements/prod"
+    chdir: "{{ arteria_service_src_path }}"
+    executable: "{{ arteria_service_env_root }}/bin/pip"
+    state: present
+    extra_args: "-U"
+
+- name: install sequencing report service
+  pip:
+    name: .
+    chdir: "{{ arteria_service_src_path }}"
+    executable: "{{ arteria_service_env_root }}/bin/pip"
+    state: present
+    extra_args: "-U"
+
+- name: download seqreports singularity images
+  ansible.builtin.shell: |
+    module load bioinfo-tools Nextflow/21.04
+    nextflow run main.nf -profile test,singularity
+  args:
+    chdir: "{{ arteria_service_src_path }}/seqreports"
+  environment:
+    NXF_SINGULARITY_CACHEDIR: "{{ nf_core_vars.NXF_SINGULARITY_CACHEDIR }}"
```

Comment:
Is it possible to access this variable without `https://docs.ansible.com/ansible/latest/collections/ansible/builtin/include_vars_module.html` or adding it as a depenency role?

Reply:
Oh yes good catch! I'll put the dependency back :+1: